client/src/components/Filter.js

```diff
@@ -225,100 +228,176 @@ export default class Filter extends Component<Props, State> {
+      languages,
+      languageSelected,
+      locations,
+      locationSelected,
+      subcategorySelected,
+      openKeys,
+    } = this.state;
 
-            <ButtonDropdown
-              isOpen={this.state.costDropdownOpen}
-              toggle={this.costToggle}
-            >
-              <DropdownToggle caret>
-                {this.state.costSelected === ''
-                  ? 'Cost'
-                  : this.state.costSelected}
-              </DropdownToggle>
-              <DropdownMenu>
-                {this.state.costs.map(value => {
+    return (
+      <Layout>
+        <div>
+          <FilterHeader
+            categorySelected={categorySelected}
+            subcategorySelected={subcategorySelected}
+          />
+          <Layout style={{ background: 'white' }}>
+            <Sider style={{ background: 'white' }}>
+              <Menu
+                mode="inline"
+                openKeys={openKeys}
+                onOpenChange={this.onOpenChange}
+              >
+                {Object.keys(categories).map(categoryName => {
                   return (
-                    <DropdownItem onClick={() => this.costSelect(value)}>
-                      {value}
-                    </DropdownItem>
+                    <SubMenu
+                      key={categoryName}
+                      title={categoryName}
+                      onTitleClick={() => this.categorySelect(categoryName)}
+                    >
+                      {categories[categoryName].map(subCategory => {
+                        return (
+                          <Menu.Item
+                            key={subCategory}
+                            onClick={() => this.subcategorySelect(subCategory)}
+                          >
+                            {subCategory}
+                          </Menu.Item>
+                        );
+                      })}
+                    </SubMenu>
                   );
                 })}
-              </DropdownMenu>
-            </ButtonDropdown>
-          </div>
-
-          <div className="dropdowns-container">
-            {Object.keys(this.state.categories).map(categoryName => {
-              return (
-                <FilterCategory
-                  categoryName={categoryName}
-                  subcategories={this.state.categories[categoryName]}
-                  categoryClickHandler={this.categorySelect}
-                  subcategoryClickHandler={this.subcategorySelect}
-                />
-              );
-            })}
-          </div>
-
-          <div className="filter-preview-container">
-            {this.state.filteredResources.map(value => (
-              <FilterPreview
-                key={value.name}
-                resourceName={value.name}
-                id={value._id}
-              />
-            ))}
-          </div>
+              </Menu>
+            </Sider>
+            <Content style={{ maxHeight: '65vh', overflowY: 'scroll' }}>
```

Comment:
Okay for now, but soon we should try to move all inline styling to external files